lib/index.js

```diff
@@ -38,8 +38,11 @@
  *   dash;
  *   when `'inverted'`, turns three dashes into an en dash and two into an em
  *   dash.
- * @property {boolean | null | undefined} [ellipses=true]
+ * @property {'unspaced' | 'spaced' | boolean | null | undefined} [ellipses=true]
  *   Transform triple dots (default: `true`), with or without spaces between.
+ *   when `true`, turns triple dots into ellipses (with or without spaces between);
+ *   when `'unspaced'`, turns triple dots into ellipses (only for those without spaces);
+ *   when `'spaced'`, turns triple dots into ellipses (only for those with spaces).
```

Comment:
```suggestion
 *   Transform triple dots (default: `true`).
 *   when `'spaced'`, turns triple dots with spaces into ellipses;
 *   when `'unspaced'`, turns triple dots without spaces into ellipses;
 *   when `true`, turns triple dots with or without spaces into ellipses.
```

Sorting, move the stuff about spaces to the input instead of output